release-notes/aws/v9.0.1.md

```diff
@@ -39,3 +39,18 @@ Below, you can find more details on components that were changed with this relea
 - Make `autopath` plugin configurable, optional and disabled by default.
 - Allow custom forward configuration destination and options.
 - Add Pod Disruption Budget.
+
+### kube-state-metrics v1.9.5 ([Giant Swarm app v1.0.5](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v105))
+
+- Updated from `kube-state-metrics` v1.9.2 for some stability fixes.
+- Removed addon resizer sidecar container.
+- Removed CPU and memory limits.
+
+### nginx-ingress-controller v0.30.0 ([Giant Swarm app v1.6.4](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v164-2020-03-17))
+
+- Enabled HorizontalPodAutoscaler by default, accompanied with following configuration defaults changes:
+  - CPU resource requests changed from 500m to 2 CPU
+  - Memory requests changed from 600Mi to 2.5GB
+  - Number of NGINX worker processes changed from 4 to 1
```

Comment:
```suggestion
### nginx-ingress-controller v0.30.0 ([Giant Swarm app v1.6.5](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v165-2020-03-23))

- Enabled HorizontalPodAutoscaler by default for selected cluster profiles.
```

Align with communicated details on this change so far in other releases like kvm 11.2.1 https://github.com/giantswarm/releases/blob/master/release-notes/kvm/v11.2.1.md